dynawo/sources/API/CRV/CRVXmlImporter.cpp

```diff
@@ -57,6 +57,8 @@ XmlImporter::importFromStream(std::istream& stream) const {
   try {
     bool xsdValidation = false;
     if (getEnvVar("DYNAWO_USE_XSD_VALIDATION") == "true") {
+      if (!hasEnvVar("DYNAWO_XSD_DIR"))
```

Comment:
Why not factorizing the test inside getEnvVar method?

Reply:
If we do this all environment variables become mandatory and they are not at the moment. Or we could have to methods getEnvVar and getMandatoryEnvVar I guess

Reply:
I agree with this proposal

Reply:
ok i'll do that